suites/test_publish_module.py

```diff
@@ -166,3 +177,20 @@ def product_info_on_customer_portal(self):
                 self.driver.close()
                 utilities.switch_to_first_tab(self.driver)
 
+    def teardown_suite(self):
+        response = unpublish_module(self, constants.module_to_unpublish, constants.variant)
+        check_that("Unpublish request status code", response.status_code, equal_to(200))
+        lcc.log_info("Module published for above test is unpublished successfully..")
+
+
+def unpublish_module(self, module_path, variant):
```

Comment:
instead of adding this method as a part of a test, it would make more sense to add it in utilities